packages/monorepo-builder/packages/split/tests/Process/ProcessFactoryTest.php

```diff
@@ -25,6 +25,7 @@ protected function setUp(): void
     public function test(): void
     {
         $subsplitProcess = $this->processFactory->createSubsplit(
+            'master',
```

Comment:
Why is this needed?

Reply:
My idea was to always have to explicitly define the branch you want to split for. The only place it is optional is when running the command the `branch` option is optional but have a default value to `master`.

I assumed that the only entrypoint was the command so the `branch` value is initiated there and carried along the process.

If you want I can change that and make the `branch` parameter optional everywhere and default to `master`, I just thought that having a unique place to define the default value for it was a good thing.

@TomasVotruba let me know

Reply:
I see. looks good :+1: